keanu-project/src/test/java/io/improbable/keanu/vertices/bool/nonprobabilistic/operators/multiple/BoolConcatenationVertexTest.java

```diff
@@ -28,38 +29,38 @@ public void canConcatVectorsOfDifferentSize() {
         ConstantBoolVertex a = new ConstantBoolVertex(new boolean[]{true, true, true});
         ConstantBoolVertex b = new ConstantBoolVertex(new boolean[]{false, false, false, false, false, false});
 
-        BoolConcatenationVertex concatZero = new BoolConcatenationVertex(1, a, b);
+        BoolConcatenationVertex concatZero = new BoolConcatenationVertex(0, a, b);
```

Comment:
you concatenate on a different dimension now?

Reply:
There's no longer a dimension 1. Concat along dimension 0 for vectors.

Reply:
ok